demos/rag_agentic/notebooks/Level1_foundational_RAG.ipynb

```diff
@@ -66,15 +54,15 @@
   },
```

Comment:
`"Next, we will initialize our environment as described in detail in our [\"Getting Started\" notebook](demos/rag_agentic/notebooks/Level0_getting_started_with_Llama_Stack.ipynb). Please refer to it for additional explanations."` 

Link not working!